app/routes/definitions.py

```diff
@@ -356,6 +356,52 @@ async def create_credential_definition(
                     rev_reg_id=revoc_reg_creation_result.revoc_reg_id, state="active"
                 )
                 credential_definition_id = active_rev_reg.result.cred_def_id
+                try:
+                    async with client_from_auth(auth) as aries_controller:
+                        bound_logger.debug(
+                            "Writing firts accum value to ledger (rev_reg_entry)"
+                        )
+
+                        await aries_controller.revocation.publish_rev_reg_entry(
+                            rev_reg_id=revoc_reg_creation_result.revoc_reg_id,
+                            conn_id=endorser_connection.results[0].connection_id,
```

Comment:
here we can deduplicate because same field is read above:
```py
await publish_revocation_registry_on_ledger(
                    controller=aries_controller,
                    revocation_registry_id=revoc_reg_creation_result.revoc_reg_id,
                    connection_id=(
                        endorser_connection.results[0].connection_id
                        if has_connections
                        else None
                    ),
                    create_transaction_for_endorser=has_connections,
                )
```

Note that the connection_id might be None if `has_connections` = 0. All issuers will have a connection with an endorser (from their onboarding) but it's worth asking if this method works if conn_id is None, and if a exception handling should be wrapped around it